src/breadcrumb.scss

```diff
@@ -83,10 +83,8 @@ $block: #{$fd-namespace}-breadcrumb;
 
     @include fd-link();
 
```

Comment:
See with designers if there needs to be a max width, if not it is good! If so apply it with overflow!